django_loose_fk/fields.py

```diff
@@ -186,11 +188,11 @@ def max_length(self) -> Union[None, int]:
 class FkOrURLDescriptor:
     field: FkOrURLField
 
-    @property
+    @cached_property
     def fk_field_name(self) -> str:
         return self.field._fk_field.name
 
-    @property
+    @cached_property
     def url_field_name(self) -> str:
         return self.field._url_field.name
```

Comment:
caching the properties twice could maybe cause issues?

Reply:
I don't think it would be an issue, because these field names (as well as the underlying fields) don't change at runtime, they are fixed for a `FkOrURLField` instance. I think I'll remove them however, since these are the equivalent of a `getattr` operation, so it's not really worth caching. I'll keep the cache on the fields